### DISCUSSION TITLE
Multiple screen() queries

### DISCUSSION THREAD

**Discussion by badg0003** [tailwindlabs/tailwindcss#12430] [Help] [answered]
Utilizing the `screen()` function is great, but I'm not seeing examples of how I can have a `max-width` constraint. AFAIK you can only provide a `min-width` like so: 

```css
@media screen(lg) {

}
```

However, what if I want to contain this to be between `sm` and `lg`?

**Comment by alex-krasikau** [comment, marked as answer]
Hey @badg0003!

First you need to add `max-lg` utility:
```
export default {
  theme: {
    extend: {
      screens: {
        'max-lg': {'max': '1023px'},
      }
    },
  },
}
```

Than you can use the following syntax:
```
@media screen(sm) and screen(max-lg) {
  /* Your styles */
}
```

**Comment by abubakar2029** [comment]
> Utilizing the `screen()` function is great, but I'm not seeing examples of how I can have a `max-width` constraint. AFAIK you can only provide a `min-width` like so: 
> 
> ```css
> @media screen(lg) {
> 
> }
> ```
> 
> However, what if I want to contain this to be between `sm` and `lg`?  

There is option of "max-lg:" it will act as max-width-1024px.
